### ISSUE TITLE
The learning rate has not declined

### ISSUE THREAD

**Issue by CCaoyu** [junyanz/pytorch-CycleGAN-and-pix2pix#574]
I know the learning rate should gradually decrease after 100 epoch, but it always 0.0002. why?
![0000](https://user-images.githubusercontent.com/44216673/54250124-840a3200-457d-11e9-8268-788fd536ddd4.PNG)

**Comment by getamu** [comment]
Did you find the answer?

**Comment by junyanz** [comment]
It should be fixed with the latest commit. Please check out the code again.

**Comment by CCaoyu** [comment]
It works. Thanks.

**Comment by AshuKV** [comment]
> It should be fixed with the latest commit. Please check out the code again.

But how should it be done?

**Comment by junyanz** [comment]
I updated the code. Just check out the latest code.
